packages/client/base/src/types/index.ts

```diff
@@ -1,6 +1,7 @@
 export * from "./errors";
 export * from "./events";
 export * from "./hosted";
+export * from "./hosted/verification";
```

Comment:
Typically this export should go in /hosted/index.ts

So just export * from ./verification